README.md

```diff
@@ -1,4 +1,4 @@
-Jasmine-Node Helpers
+Jasmine Node Helpers
```

Comment:
This and other places where the repo is mentioned in the Readme have been updated.